content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -9,7 +9,7 @@ weight: 30
 Operators are software extensions to Kubernetes that make use of
 [custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
 to manage applications and their components. Operators follow
-Kubernetes principles, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+Kubernetes principles, notably the [control loop](/docs/reference/glossary/?all=true#term-control-plane).
```

Comment:
This change could use the `control plane` link listed above or the glossary term.

Reply:
```suggestion
Kubernetes principles, notably the [control loop](/docs/concepts/architecture/controller/).
```